packages/login-form/tsconfig.build.json

```diff
@@ -5,5 +5,5 @@
     "rootDir": "./src",
     "baseUrl": "./"
   },
-  "include": ["./src"]
+  "include": ["./src/**/*"]
```

Comment:
Won't it generate declarations in a form `packages/input/*` and `packages/login-form` instead of typedefs for this package only?

Reply:
leftover, but in general no differences as 'src' and 'src/**/*' are actually same - load everything inside 'src'. Fixed